src/commcare_cloud/terraform/modules/postgresql/main.tf

```diff
@@ -41,10 +52,10 @@ module "postgresql" {
   subnet_ids = "${var.subnet_ids}"
 
   # DB parameter group
-  family = "postgres9.6"
+  family = "postgres${local.computed_major_engine_version}"
 
   # DB option group
-  major_engine_version = "9.6"
+  major_engine_version = major_engine_version = local.major_engine_version == "" ? local.computed_major_engine_version : var.major_engine_version
```

Comment:
Should `major_engine_version = major_engine_version = ` be just `major_engine_version = `? I think this probably needs to be changed but let me know if I'm misunderstanding